.env.template

```
@@ -1,2 +1,2 @@
-DATABASE_URL="postgresql://postgres:password@localhost:5432/hetic?schema=public"
-ETHERSCAN_API_KEY=
\ No newline at end of file
+DATABASE_URL=postgresql://postgres:password@localhost:5432/hetic?schema=public
+ETHERSCAN_API_KEY=R7BT5JU6PJ6ED6A62V56NXEZYP5G8N9W8P
```

Bon bah gg faut changer la clé API 🤡 
```suggestion
ETHERSCAN_API_KEY=
```